Docs/source/install/hpc/summit.rst

```diff
@@ -125,8 +125,8 @@ Use the following :ref:`cmake commands <building-cmake>` to compile:
    rm -rf build_summit
 
    cmake -S . -B build_summit -DWarpX_COMPUTE=CUDA -DWarpX_PSATD=ON -DWarpX_QED_TABLE_GEN=ON -DWarpX_PYTHON=ON -DWarpX_DIMS="1;2;RZ;3"
-   cmake --build build_summit -j 16
-   cmake --build build_summit -j 16 --target pip_install
+   cmake --build build_summit -j 8
+   cmake --build build_summit -j 8 --target pip_install
```

Comment:
Alternatively we can prefix these with `runNode` as done above for the dependencies.

Potentially needs to wait a bit for a free node, but would be good practice.
```suggestion
   runNode cmake --build build_summit -j 24
   runNode cmake --build build_summit -j 24 --target pip_install
```